src/pages/Task.tsx

```diff
@@ -82,9 +88,21 @@ export const Task: React.FC = () => {
         versionId={version}
         patchNumber={patchNumber}
       />
-      <PageHeader>
-        <H1>Current Task Name</H1>
-      </PageHeader>
+      {loading ? (
+        <PageHeader>
+          <Skeleton active={true} paragraph={{ rows: 0 }} />
+        </PageHeader>
+      ) : displayName || status ? (
+        <PageHeader>
+          <H2 id="task-name">{displayName}</H2>
+          {"  "}
+          <BadgeWrapper>
+            <ErrorBoundary>
```

Comment:
Just curious - is there a specific reason you're adding an error boundary around this component?

Reply:
TaskStatusBadge throws an error
https://github.com/evergreen-ci/spruce/pull/113/files#diff-a6da40343c81cd82a8f283202322bfc9R62**